model/application_category.go

```diff
@@ -90,7 +90,9 @@ var BuiltinCategoryPatterns = map[ApplicationCategory][]string{
 		"*/*vmselect*",
 		"*/*vmstorage*",
 		"*/*vmagent*",
+		"*/*vmalert*",
 		"datadog/*",
 		"*/*datadog*",
+		"*/*karma*",
```

Comment:
Matching `karma` seems too broad. Some legit apps might use `karma` in their name, like Reddit infra :)

Reply:
I agree with you :) I fixed it